style.css

```diff
@@ -6,8 +6,6 @@
   padding: 0;
 }
 
-/* outline: 1px solid red; */
-
 body {
```

Comment:
- [ OPTIONAL] Good job implementing the changes requested by the previous reviewer :clap: , kindly make sure you use the right background color to match the Figma design. Yours is darker than the Figma's.
![Screenshot from 2023-01-27 01-41-21](https://user-images.githubusercontent.com/97834160/214975499-7579e00e-5870-417d-875c-fe0595ff819d.png)